app/assets/config/manifest.js

```diff
@@ -1,3 +1,3 @@
 //= link_tree ../images
 //= link_directory ../javascripts .js
-//= link_directory ../stylesheets .css
+//= link application.css
```

Comment:
Could this change be the culprit? Upstream, I'm still seeing: https://github.com/rails/rails/blob/v6.1.1/railties/lib/rails/generators/rails/app/templates/app/assets/config/manifest.js.tt